src/collective/collectionfilter/filteritems.py

```diff
@@ -9,7 +9,12 @@
 from collective.collectionfilter.vocabularies import DEFAULT_FILTER_TYPE
 from collective.collectionfilter.vocabularies import EMPTY_MARKER
 from Missing import Missing
+from plone.app.blocks.layoutbehavior import ILayoutBehaviorAdaptable
```

Comment:
I just tried your branch: if you do not have installed plone.app.mosaic (which is no hard dependency) this import breaks ... I'd put it into the try/except block

Reply:
Sorry I fixed that but perhaps in a different branch. easy fix